units/deco_host_notify.go

```diff
@@ -134,7 +136,10 @@ func (j *decoHostNotifyJob) Run(_ context.Context) {
 		Summary:     fmt.Sprintf("investigate automatically decommissioned host '%s'", j.Host.Id),
 		Type:        "Incident",
 		Description: strings.Join(descParts, "\n"),
-		Components:  []string{"Evergreen"},
+		Components:  []string{"Static Host Management"},
+		Fields: map[string]interface{}{
+			fieldAssignedTeams: []map[string]string{{"id": runtimeEnvironmentsId}}, // Assigned Teams: Runtime Environments
```

Comment:
I thought you had said "value", not "id", in Slack.

Reply:
@brooookemiller mentioned that its actually id since the value we are using for runtimeEnvironments is an id

Reply:
I had no idea we made tickets this way. It looks like we could set up a Splunk alert to create a ticket like this which I think would be way better than hardcoding it. I get the sense that this is a quick fix to get us over to the DEVPROD project but could you make a follow up ticket?

Reply:
Yes that was the intent behind this. Great idea! https://jira.mongodb.org/browse/DEVPROD-2486